src/utils/typeStrings/typeString_grammar.pegjs

```diff
@@ -0,0 +1,482 @@
+Start =
```

Comment:
Why are some scripts related to string parser in Warp now? Weren't they defined in solc-typed-ast?

Reply:
Because they were removed from `solc-typed-ast` in favor of InferType class. The InferType class however seems to be so different from the typestring parser that it requires a lot of effort to port warp from the old type inference system to the new one. Thus this partial port and the plan to adjust warp to the usage of InferType gradually